Reqnroll/Formatters/Configuration/EnvVariableEnableFlagParser.cs

```diff
@@ -0,0 +1,21 @@
+﻿using Reqnroll.CommonModels;
+using Reqnroll.EnvironmentAccess;
+
+namespace Reqnroll.Formatters.Configuration;
+
+public class EnvVariableEnableFlagParser : IEnvVariableEnableFlagParser
+{
+    private readonly IEnvironmentWrapper _environmentWrapper;
+
+    public EnvVariableEnableFlagParser(IEnvironmentWrapper environmentWrapper)
+    {
+        _environmentWrapper = environmentWrapper;
+    }
+
+    public bool Parse()
+    {
+        var enabledVariable = _environmentWrapper.GetEnvironmentVariable(FormattersConfigurationConstants.REQNROLL_FORMATTERS_ENABLE_ENVIRONMENT_VARIABLE);
+        var enabledVariableValue = enabledVariable is Success<string> envVarSuccess && bool.TryParse(envVarSuccess.Result, out bool result) ? result : true;
```

Comment:
(6) The `true` as a default value looked strange. Essentially what we have is not enabling, but disabling (opt out). Maybe we can just add a comment about this for now.

Reply:
Yes, I understood it at the end. Maybe we should call it "disable" instead of "enable" then for better clarity (and negate the values).